nixos/tests/xrdp-with-audio-pipewire.nix

```diff
@@ -0,0 +1,89 @@
+# This test is expected to fail at the end because Pipewire can't run
```

Comment:
What is the value of this test if it is expected to fail? I think this test can be removed and added back if we ever add Pipewire support.

Reply:
I think so. What if I also integrate pipewire-module-xrdp?

Reply:
You can go either way, but I would prefer to have it in a separate PR to avoid getting in an infinite rabbit hole and never getting the PR merged.

Reply:
I remove the test

Me, or someone else can add something like this when integrating pipewire-module-xrdp.